components/base/Header.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
+import { ElMenu, ElSwitch } from "element-plus";
```

Comment:
have you tried this to avoid manual importing? 
https://element-plus.org/en-US/guide/quickstart.html#on-demand-import